src/main/java/com/redhat/devtools/lsp4ij/features/rename/LSPRenameRefactoringDialog.java

```diff
@@ -192,4 +221,32 @@ protected void dispose() {
         super.dispose();
     }
 
+    @NotNull
+    private static Set<PsiReference> getExternalReferences(@NotNull PsiFile file, int offset) {
+        Set<PsiReference> externalReferences = new LinkedHashSet<>();
+
+        // When testing, just collect references on the current thread
+        if (ApplicationManager.getApplication().isUnitTestMode()) {

```

Comment:
This is consistent with the existing precedent of keeping things a bit simpler when tests execute.